manifests/ovs-cni.yml.in

```diff
@@ -36,7 +37,78 @@ spec:
         volumeMounts:
         - name: cnibin
           mountPath: /host/opt/cni/bin
+      - name: ovs-cni-marker
+        image: ${OVS_CNI_MARKER_IMAGE_REPO}/${OVS_CNI_MARKER_IMAGE_NAME}:${OVS_CNI_MARKER_IMAGE_VERSION}
+        imagePullPolicy: ${OVS_CNI_MARKER_IMAGE_PULL_POLICY}
+        securityContext:
+          privileged: true
+        command:
+          - sh
+          - -c
+          - |
+            if [ -e /host/var/run/openvswitch/db.sock ]; then
+              echo "openvswitch running on host"
+              ovs_socket="unix:///host/var/run/openvswitch/db.sock"
+            elif [ -e /host/run/openvswitch/db.sock ]; then
+              echo "openvswitch taken from openshift"
+              ovs_socket="unix:///host/run/openvswitch/db.sock"
+            else
+              exit 1
+            fi
+            /marker -v 3 -logtostderr -node-name $(NODE_NAME) -poll-interval 10 -ovs-socket ${ovs_socket}
```

Comment:
Should poll-interval=10 be then the default value for the flag in code so we can avoid setting it here?

Reply:
Fair enough